### PR TITLE
[PD-2631] fixed discrepencies from Jason Sole's QC

### PR DESCRIPTION
**Pull request by huy-nguyen-colorado**
The  "Companies" sub-menu under "Employers" in topbar nav was not visible when I built out the new topbar. Changed my Company permission in the dkm shell so I can see this in my local topbar, then fixed the problem.  Added v2 version of topbar.js because of companies in the topbar nav, removed mobile JS that are not needed for this PR or maybe ever.  Removed un-used legacy topbar styles.

To test:
1. Please ensure you have access to the "Companies" submenu under Employers in tob bar nav.
2. Please ensure  that the "DirectEmployer Association" option is in the sub-menu and that this option also contain its own sub-menu.

### CODE REVIEW

**Comment by koonsies** [comment]
Small nitpick.. You have one branch called "PD-2631-topbar-update" and one called "PD-2631-topbar-updates".. At first, I had checked out the incorrect branch. May want to make sure the branches (if different) have a bit more variance in naming haha.

File: gulp/src/sass/seo_base_styles.scss [hunk 1]
```diff
@@ -13,6 +13,8 @@ $red-maroon: #954343;
 $black: #000000;
 $white: #ffffff;
 
+
+
```
Comment (koonsies) [inline gulp/src/sass/seo_base_styles.scss]: Extra white space?

Comment (huy-nguyen-colorado) [inline gulp/src/sass/seo_base_styles.scss]: Not necessary, was added during merge, I will remove muy pronto.

File: gulp/src/sass/seo_base_styles.scss [hunk 2]
```diff
@@ -156,7 +158,7 @@ a.direct_optionsLess {
   }
 
   .menu-container > ul > li:last-of-type > ul {
-      left: 0;
+      right: 0;
```
Comment (huy-nguyen-colorado) [inline gulp/src/sass/seo_base_styles.scss]: Yeah, I changed that because Jason Sole wanted me to.

Comment (huy-nguyen-colorado) [inline gulp/src/sass/seo_base_styles.scss]: I originally had the last of type in the navbar on the left, but Jason wanted it on the right.

Comment (koonsies) [inline gulp/src/sass/seo_base_styles.scss]: I gotcha. We'll move forward with this styling, then.

**Pull request merged**